docs/src/pages/guide.md

```diff
@@ -0,0 +1,94 @@
+# Getting Started
+
+`Nabla.jl` has two interfaces, both of which we expose to the end user. We first provide a minimal working example with the low-level interface, and subsequently show how the high-level interface can be used to achieve similar results. More involved examples can be found [here](https://github.com/invenia/Nabla.jl/tree/master/examples).
```

Comment:
The order below is reversed; you show the high-level interface and then the low-level interface.